src/pages/ProductList/index.js

```diff
@@ -0,0 +1,70 @@
+import { Container, ProductsGrid } from "./../../styled-components";
+import { useState, useEffect } from "react";
+import { createProductAdapter } from "./../../adapters/featured-products";
+import Products from "../../mocks/en-us/products.json";
+import {
+  ProductSidebarFilter,
+  ProductCard,
+  PageControlls,
+} from "./../../components/product";
+import { HeadingPage, WrapperContent, Content } from "./styled";
+import { getFilterProducts } from "./../../utils/getFilterProducts";
+import Spinner from "../../components/ui/Spinner/Spinner";
+import { FilterButton } from "./../../components/common";
+
+const ProductList = () => {
+  const [filters, setFilter] = useState([]);
+  const [productList, setProductList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [showFilter, setShowFilter] = useState(false);
+
+  let loadingInterval;
+
+  useEffect(() => {
+    load();
+    return () => clearInterval(loadingInterval);
```

Comment:
nice, i like that you apply the clear but...is interval o timeout what the code should implement?

Reply:
I don't think so, but it's an easiest way to implement the requested, maybe a better way is with promises